### PR TITLE
Update last tests for first version of test specification

### LINKED ISSUE #606: Update all tests for test specification
Test name, description, test.step() and the logical topology needs to be modified to that the final specification looks nice. In the testing.md it is described how to write them.

This is since the introduction of automatic generated test specification in #604

- [x] ietf-system (@mattiaswal)
- [x]  Merge infix-interfaces into ietf-interface (@mattiaswal)
- [x] ietf-interfaces (@mattiaswal)
- [x] ietf-hardware
- [x] ietf-routing (@mattiaswal)
- [x] ietf-syslog (@troglobit)
- [x] infix-containers (@troglobit)
- [x] infix-dhcp (@troglobit)
- [x] infix-services (@troglobit)
- [ ] meta
- [x] misc

### PR DESCRIPTION
**Pull request by mattiaswal**
## Description

Fix #606 
## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [ ] Bugfix
  - [ ] Regression tests
  - [ ] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: test/case/infix_containers/container_bridge/test.py [hunk 1]
```diff
@@ -84,18 +84,18 @@
...
-        _, hport = env.ltop.xlate("host", "data")
-        url = infamy.Furl(URL)
+    _, hport = env.ltop.xlate("host", "data")
+    url = infamy.Furl(URL)
 
-        with infamy.IsolatedMacVlan(hport) as ns:
-            ns.addip(OURIP)
+    with infamy.IsolatedMacVlan(hport) as ns:
+        ns.addip(OURIP)
+        with test.step("Verify host:data can ping 10.0.0.2"):
```
Comment (troglobit) [inline test/case/infix_containers/container_bridge/test.py:96]: Maybe "Verify basic DUT connectivity, host:data can ping DUT 10.0.02" or something?  It's not for the container, so not to confuse the reader what this step is for.

File: test/case/infix_containers/container_bridge/test.py [hunk 2]
```diff
@@ -84,18 +84,18 @@
...
             ns.must_reach(DUTIP)
-
+        with test.step("Verify container 'web-docker0' is reachable on http://10.0.0.2:91"):
```
Comment (troglobit) [inline test/case/infix_containers/container_bridge/test.py:98]: Wrong port, "Verify container web server 'web-docker0' is reachable on http://10.0.0.2:8080"

Also, maybe we should mention somewhere this is a port forward test?

File: test/case/infix_containers/container_veth/test.py [hunk 3]
```diff
@@ -86,18 +86,19 @@
             }
         })
 
-    with test.step(f"Verify {NAME} container has started"):
+
```
Comment (troglobit) [inline test/case/infix_containers/container_veth/test.py:89]: Extra linefeed crept in.

**Review by troglobit — CHANGES_REQUESTED** [review]
Nice work!  I see now the container tests are really difficult to document, e.g., in comparison to DHCP client.

For the "test all possible interface combinations" test, @ahmkar94, me and wkz talked about how to describe networks that are internal to the DUT, like VETH pairs and bridges, that have no physical counterpart or connection to the host -- i.e., that we cannot describe in the topology.dot file.   One option, that somehow was abandoned, was an [ASCII art picture](https://github.com/kernelkit/infix/blob/main/test/case/ietf_interfaces/verify_all_interface_types/test.py) included in the top test docstring that would be translated to the Readme.adoc file.  Maybe the container tests would benefit even more from this?

**Example:**

```
  .-------------.         .---------------.     .--------.
  |      |  tgt |---------| mgmt |        |     |  web-  |
  | host | data |---------| data | target |     | server |
  '-------------'         '---------------'     '--------'
                             |                    /
                            br0                  /
                              `----- veth0 -----'
```

**Comment by mattiaswal** [comment]
> Nice work! I see now the container tests are really difficult to document, e.g., in comparison to DHCP client.
> 
> For the "test all possible interface combinations" test, @ahmkar94, me and wkz talked about how to describe networks that are internal to the DUT, like VETH pairs and bridges, that have no physical counterpart or connection to the host -- i.e., that we cannot describe in the topology.dot file. One option, that somehow was abandoned, was an [ASCII art picture](https://github.com/kernelkit/infix/blob/main/test/case/ietf_interfaces/verify_all_interface_types/test.py) included in the top test docstring that would be translated to the Readme.adoc file. Maybe the container tests would benefit even more from this?
> 
> **Example:**
> 
> ```
>   .-------------.         .---------------.     .--------.
>   |      |  tgt |---------| mgmt |        |     |  web-  |
>   | host | data |---------| data | target |     | server |
>   '-------------'         '---------------'     '--------'
>                              |                    /
>                             br0                  /
>                               `----- veth0 -----'
> ```

Yes, it has been a struggle, An extra ASCII art I think will do it more understandable. But for now this will do, but of course you are right. We need do describe software-ports in a better way.

File: test/case/infix_containers/container_bridge/test.py (hunk 2 above)
Comment (mattiaswal) [reply test/case/infix_containers/container_bridge/test.py:98]: Sorry, embarrassing mistake, will fix that.

**Review by troglobit — APPROVED** [review]

**Pull request merged**